### DISCUSSION TITLE
［委托］治理鬼火少年

### DISCUSSION THREAD

**Discussion by abyssal-sea-c** [share121/inter-knot#5462] [委托]
今天在人行道上走的好好的被超载鬼火创了 希望治安局治理横冲乱撞的鬼火喵
![IMG_20240928_193453](https://github.com/user-attachments/assets/f527d535-1f8f-4751-93f4-3f92e6195a5e)

**Comment by Firefly704** [comment]
真是可恶，前几天去光映广场还差点被鬼火撞上，还好有个机巧治安官把那辆车截停了，不然这会儿我也在医院躺着了

**Reply by abyssal-sea-c** [reply]
我是真被鬼火创了喵

**Reply by Firefly704** [reply]
抱抱，楼主看起来像是胳膊被创了，这几天要按时敷药养伤，千万不要感染了喵

**Comment by LiuZhy2004** [comment]
每一个鬼火都迟早会遇到属于自己的泥头车

**Comment by Orangered81** [comment]
这就把白祈重工的大运开过来创飞<img src="https://img-static.mihoyo.com/communityweb/upload/068a8b611fae7cde48470ebfe5e21847.png" alt="米游姬得意" width="75" >

**Comment by Ayn1631** [comment]
鬼火

**Comment by Tengzhenlian** [comment]
这个委托我接了！

**Comment by meromerome** [comment]
希望人没事

**Comment by TianyiLuo0721** [comment]
我有卡吕东之子的电话，我叫他们来帮帮忙

**Comment by c328422** [comment]
你很幸運沒骨折，我的認識一個人騎腳踏車被重機追撞，大腿骨粉碎性骨折，目前整隻腳必須強制修養
